src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/service/JWKKeyAccessor.kt

```diff
@@ -29,6 +29,7 @@ class JWKKeyAccessor(
 ) {
   private val primaryKeyStore: KeyStore = initializePrimaryKeyStore()
   private val auxiliaryKeyStore: KeyStore = initializeAuxiliaryKeyStore()
+
```

Comment:
Sorry ignore changes in this file, just whitespace and I moved the getKeyPair private method down a bit as I was reading through it again.